client/joinus_client/src/components/Events/EventCard.js

```diff
@@ -55,15 +45,11 @@ export default function EventCard(props) {
     showUserEvents,
     joinedEvents,
     setReload,
-    reload
+    reload,
```

Comment:
Extra comma here- I think this might break the build? (kinda cray to think about tbh)